### PR TITLE
Unsteady CHT tutorial

### PR DESCRIPTION
**Pull request by oleburghardt**
This PR adds a follow-up on [Heated Cylinders with Conjugate Heat Transfer](https://su2code.github.io/tutorials/Inc_Heated_Cylinders/) to the tutorials catalogue.

It is about how to turn a steady simulation into an unsteady one within the multizone framework, based on CHT but not exclusively applicable for such cases.

As it's a rather small one, @talbring considers to include some more specifics on his work on paraview multiblock files.

This tutorial relates to the [Unsteady CHT pull request](https://github.com/su2code/SU2/pull/854).

### CODE REVIEW

File: Inc_Heated_Cylinders_Unsteady/cht_2d_3cylinders.cfg [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+%
+MARKER_CHT_INTERFACE= (cylinder_outer1, cylinder_inner1, cylinder_outer2, cylinder_inner2, cylinder_outer3, cylinder_inner3)
+%
+%
+TIME_DOMAIN = YES
+%
+%
+TIME_MARCHING= DUAL_TIME_STEPPING-2ND_ORDER
+%
+%
```
Comment (TobiKattmann) [inline Inc_Heated_Cylinders_Unsteady/cht_2d_3cylinders.cfg]: See my comment in the related PR https://github.com/su2code/SU2/pull/854

File: _tutorials/incompressible_flow/Inc_Heated_Cylinders_Unsteady.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+title: Unsteady Conjugate Heat Transfer
+permalink: /tutorials/Inc_Heated_Cylinders_Unsteady/
+---
+
+<br>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/MqN8GalSyzk" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (TobiKattmann) [inline _tutorials/incompressible_flow/Inc_Heated_Cylinders_Unsteady.md]: 👍

File: _tutorials/incompressible_flow/Inc_Heated_Cylinders_Unsteady.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+The following tutorial will walk you through the steps required when solving for an unsteady coupled CHT solution. It is assumed you have already obtained and compiled the SU2_CFD code for a serial computation. If you have yet to complete these requirements, please see the [Download](/docs/Download/) and [Installation](/docs/Installation/) pages and that make sure you have completed the [heated cylinders with conjugate heat transfer tutorial](/tutorials/Inc_Heated_Cylinders/).
+
+### Background
+
+For unsteady flows around walls that are transferring heat from an adjacent (solid) zone, the coupling of temperature and heat flux distributions has to be resolved for each and every time step. Both will vary over time as they depend on the current flow field.
+
+### Problem Setup
+
+The problem setup is the same as in the [heated cylinders with conjugate heat transfer tutorial](/tutorials/Inc_Heated_Cylinders/) except for the density. It is increased in all zones by a factor of 100 so that for the flow we obtain a Reynolds number of 4000 which will make it unsteady. Thus we set
```
Comment (TobiKattmann) [inline _tutorials/incompressible_flow/Inc_Heated_Cylinders_Unsteady.md]: `...which will make it unsteady.` sounds a bit weird albeit true

File: _tutorials/incompressible_flow/Inc_Heated_Cylinders_Unsteady.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+<iframe width="560" height="315" src="https://www.youtube.com/embed/MqN8GalSyzk" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Goals
+
+This tutorial is a follow-up on the [heated cylinders with conjugate heat transfer tutorial](/tutorials/Inc_Heated_Cylinders/) where a steady CHT solution was computed for a problem involving multiple physical zones.
+The following capabilities of SU2 will be showcased in this tutorial:
+
+- Time domain and time-marching config file options (plus related ones) for unsteady simulations
+- Use of time iterations, outer and inner iterations
+- Paraview multiblock output
```
Comment (TobiKattmann) [inline _tutorials/incompressible_flow/Inc_Heated_Cylinders_Unsteady.md]: This point should get some explanation below as it is mentioned here but never down below

File: _tutorials/index.md [hunk 5]
```diff
@@ -63,10 +63,11 @@ Simulation of external, viscous, incompressible flow around the NACA 0012 using
 Simulation of internal, laminar, incompressible flow over a backward-facing step with an inlet velocity profile input from file.
 * [Laminar Buoyancy-driven Cavity](/tutorials/Inc_Laminar_Cavity/)    
 Simulation of internal, laminar, incompressible flow in a differentially-heated cavity under the influence of gravity (classical natural convection case).
-* [Streamwise Periodicity](/tutorials/Inc_Streamwise_Periodic/)
+* [Streamwise Periodicity](/tutorials/Inc_Streamwise_Periodic/)
```
Comment (TobiKattmann) [inline _tutorials/index.md:66]: These added spaces trigger line break [here](https://su2code.github.io/tutorials/home/). Added for consistency

**Comment by TobiKattmann** [comment]
Although this could be directly merged into master (as the necessary code is already there from su2code/su2/pull/854) I think choosing develop might avoid some problems down the merging road. 

I'll test 🔍  this Tutorial a bit and then merge it if no-one (especially @oleburghardt  ;) ) has comments or thinks otherwise.

**Comment by oleburghardt** [comment]
Thanks @TobiKattmann for resurrecting this and updating the links. Did you check whether the unsteady CHT case from the tutorial repository works fine with current develop? Apart from that, I don't really remember the reason why we lost sight of this one...

Is there an official guideline how merging into master is handled for the su2code.github.io world?

**Comment by TobiKattmann** [comment]
To quote the official guideline Article 2 §4(b): "You shall merge the websites develop into master whenever the main gig (su2 itself) does so." :) The reasoning of the regulators back in the day :older_man:  was because when a feature comes with a tutorial you don't want to have the tutorial live before the feature is in master. Same goes then the tutorials folder. I guess it makes sense to step those 4 (su2, Testcases, tutorial, website) in unisono. I'll keep an eye on the next su2 master as I have that streamwise stuff in there as well ;)

I'll play around a bit with your Testcase and unsteady cht in the next days and will report :+1:

**Comment by TobiKattmann** [comment]
But that didn't really work in the past because otherwise your #49 would already  be merged :)

**Comment by oleburghardt** [comment]
> But that didn't really work in the past because otherwise your #49 would already be merged into master :)

Maybe we could bring up next Wednesday how to handle the develop branch in this and the Tutorials repository, @pcarruscag @TobiKattmann ?

**Comment by TobiKattmann** [comment]
@oleburghardt imo we can merge this rn into develop. With the next su2 master we just merge develop into master for the website and tutorials as well and we are good. If /su2code/su2/pull/1226 doesn't make it into develop over at the code we can still  quickly remove the links to it and we're good. With website, Tutorials and code togehter I guess it always is a bit wobbly :)

**Comment by Nicola-Fonzi** [comment]
Dear all,

I noticed that this PR remained open even if ready, shall we merge it?

**Comment by TobiKattmann** [comment]
I will make a few adjustments because this was meant to go in with su2code/su2/pull/1226 which is currently closed by the stall bot. Especially the config options do not have to be changed.

**Comment by TobiKattmann** [comment]
Now that su2code/su2/pull/1226 is merged this can be merged into website develop. The code PR contained change in option names that were already considered here. Thats why the merge was so delayed.

**Pull request merged**